src/index.js

```diff
@@ -0,0 +1 @@
+import './style.css';
```

Comment:
- [ ] Good job on working on the application styles but I am sorry to **inform you that your app is not working**. I tried to search for the reason and I found out that It may be due to that you may have deleted some codes here in the index.js file. Currently, on the branch `project3`, the `index.js` which acts as the entry point for bundler only contains one line of code (import ''./style.css") 
When I looked in your previous branch --> `project2` I have seen that in `index.js` , there are lot of codes that are not present in index.js on the current branch you are working in (project) 👇
![image](https://user-images.githubusercontent.com/90222110/196910321-b3d65955-3497-449d-8e9d-86dc27274b0a.png)

Please try to debug your application so that It can work well again. About the styles, they are good 👍

Reply:
Ok. Surely I am looking for that whether the previous project CSS file is not taken into consideration. Thank you very much for your review.

Reply:
I am very much thankful to you for your valuable review and guidance. I did the correction in the index.js file. Now it is working. Again I am sending the recent screenshot after the correction : 
https://user-images.githubusercontent.com/106825373/196926807-e81e7db1-efd4-468b-9dc1-f4bb57295911.png